powerstore/resource_snapshot.go

```diff
@@ -102,17 +102,9 @@ func (r *resourceVolumeSnapshot) Schema(ctx context.Context, req resource.Schema
 				},
 			},
 			"creator_type": schema.StringAttribute{
-				Optional:            true,
 				Computed:            true,
 				Description:         "Creator Type of the volume snapshot.",
 				MarkdownDescription: "Creator Type of the volume snapshot.",
-				Validators: []validator.String{
-					stringvalidator.OneOf([]string{
-						"User",
```

Comment:
should we keep validator to accept only "User" as a value.

Reply:
Since the default value is "User" only, so it does not make sense to let user enter the same. In the higher version where other values are supported then only we need choices here.

Reply:
sure, thanks.